_layouts/index.html

```diff
@@ -8,7 +8,9 @@
       <div class="group row">
         <h2 id="install">{{ t.pagecontent.install.install }}</h2>
         <br>
-        <pre style='clear:both;text-align:center;margin-bottom:0.9em'><code id='selectable' onclick="selectText(this)">/usr/bin/ruby -e &quot;$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)&quot;</code></pre>
+        <div class="highlight-wrapper" tabindex="0">
+          <pre style='clear:both;text-align:center;margin-bottom:0.9em'><code id='selectable' onclick="selectText(this)">/usr/bin/ruby -e &quot;$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)&quot;</code></pre>
```

Comment:
Can the `tabindex` be added to the `pre` directly?

Reply:
Yeah it can be set directly it would work fine. This was implemented here because using the same style definitions as other code pre elements.

Reply:
That'd be preferable, thanks!